frontend/src/employee-frontend/components/child-information/assistance-need/decision/AssistanceNeedDecisionPage.tsx

```diff
@@ -132,19 +59,31 @@ export default React.memo(function AssistanceNeedDecisionPage() {
 
   return (
     <>
-      {renderResult(
-        combine(child, assistanceNeedDecision),
-        ([child, { decision }]) => (
-          <I18nContext.Provider
-            value={{
-              lang: decision.language.toLowerCase() as Lang,
-              setLang: () => undefined
-            }}
-          >
-            <DecisionPreviewContents child={child} decision={decision} />
-          </I18nContext.Provider>
-        )
-      )}
+      <Content>
+        <ReturnButton label={i18n.common.goBack} />
+
+        <ContentArea opaque>
+          {renderResult(assistanceNeedDecision, ({ decision }) => (
+            <I18nContext.Provider
+              value={{
+                lang: decision.language.toLowerCase() as Lang,
+                setLang: () => undefined
+              }}
+            >
+              <AssistanceNeedDecisionReadOnly decision={decision} />
+            </I18nContext.Provider>
+          ))}
+        </ContentArea>
+        <Gap size="m" />
+        <CollapsibleContentArea
+          title={<H2 noMargin>{t.appealInstructionsTitle}</H2>}
+          open={appealInstructionsOpen}
+          toggleOpen={() => setAppealInstructionsOpen(!appealInstructionsOpen)}
+          opaque
+        >
+          {t.appealInstructions}
```

Comment:
This should also be within the `<I18nContext.Provider>` in order to get the Swedish version once available.